roles/edpm_ssh_known_hosts/tasks/main.yml

```diff
@@ -66,7 +66,7 @@
           {%         set _ = entries.append(hostdata[network ~ '_ip']) %}
           {%       endif %}
           {%       if (network ~ '_hostname') in hostdata %}
-          {%         set _ = entries.append(hostdata[network ~ '_hostname']) %}
+          {%         set _ = entries.append(hostdata[network ~ '_hostname'] ~ '*') %}
```

Comment:
We don't have  `ctlplane_hostname` and other `{{network}}_hostname` host vars either. I guess you've to change L75. I've also proposed https://github.com/openstack-k8s-operators/dataplane-operator/pull/462